test/Microsoft.Framework.Runtime.Tests/TestSample/project.lock.sample

```diff
@@ -0,0 +1,2180 @@
+{
```

Comment:
Will this lock file be changed if I do dnu restore in the root of the solution folder?